src/browser/provider/built-in/dedicated/chrome/cdp-client/index.ts

```diff
@@ -332,26 +334,50 @@ export class BrowserClient {
         if (!client)
             throw new Error('Cannot get the active browser client');
 
-        return this._clientFunctionExecutor.executeSelector(client.Runtime, command, callsite, selectorTimeout);
+        return this._clientFunctionExecutor.executeSelector({
+            Runtime:  client.Runtime,
+            errTypes: {
+                notFound:  Errors.CannotObtainInfoForElementSpecifiedBySelectorError.name,
+                invisible: Errors.CannotObtainInfoForElementSpecifiedBySelectorError.name,
+            },
+
+            command, callsite, selectorTimeout,
+        });
     }
 
-    public async switchToIframe (): Promise<void> {
+    public async switchToIframe (command: SwitchToIframeCommand, callsite: CallsiteRecord, selectorTimeout: number): Promise<void> {
```

Comment:
Use a single object parameter here.
`public async switchToIframe ({ command: SwitchToIframeCommand, callsite: CallsiteRecord, selectorTimeout: number }): Promise<void> {`

Reply:
I think it's unnecessary, because it's only 3 parameters.